internal/servegit/serve_test.go

```diff
@@ -32,7 +32,7 @@ func TestReposHandler(t *testing.T) {
 		repos: []string{"project1", "project2"},
 	}, {
 		name:  "nested",
-		repos: []string{"project1", "project1/subproject", "project2", "dir/project3"},
+		repos: []string{"project1", "project1/subproject", "project2", "dir/project3", "dir/project4.bare"},
```

Comment:
This doesn't seem like a hack at all, succinct. So I actually like it :) FYI I think there is a more traditional way to indicate a repo is base, and that is if the name has the suffix `.git`. I believe git used to check the suffix to decide, but it has a more robust check these days based on actual presence of git files.

Reply:
Oh yeah, The repos actually end up with `.git` in my deployment. I just felt like being explicit there.